Packs/ContentManagement/Scripts/SuggestBranchName/SuggestBranchName.py

```diff
@@ -4,6 +4,22 @@
 ATTEMPS = 10
 
 
+def find_available_branch_azure_devops(pack_name: str, command_get_branch: str):
```

Comment:
the command is the same always, please remove the argument and have it hard-coded in line 8